config/tailwind/dropdown.js

```diff
@@ -0,0 +1,43 @@
+const plugin = require('tailwindcss/plugin')
+
+module.exports = plugin(({ addComponents }) => {
+  const dropdown = {
+    '.dropdown': {
+      '@apply inline-flex flex-col gap-2 p-2 bg-white border border-gray-300 shadow min-w-[200px] max-w-[250px] rounded-xl':
```

Comment:
Shame about the automatic ordering. Do you get intellisene with @apply in these files?

Is the main benefit of creating components JS so you can have intellisense for them in your project?

Reply:
So in this case there is no real difference between using CSS or JS. Intellisense should pick up either. The reason I generally keep all of my CSS in JS is for when I do want to create ![image](https://github.com/vigetlabs/parts-kit/assets/8878152/0a08f5db-f3a0-41c1-a62d-363bab5defb2), I have that availability. And for that reason, I just like to be consistent and keep all styles in JS.

In our current system, as of now, prettier doesn't touch these styles in `@apply`, but looking at the docs for the TW plugin, it seems like there [might be some workarounds](https://github.com/tailwindlabs/prettier-plugin-tailwindcss#sorting-classes-in-template-literals).